tests/js/panelsTests.js

```diff
@@ -12,6 +12,19 @@ https://github.com/gpii/universal/LICENSE.txt
     "use strict";
 
     fluid.registerNamespace("gpii.tests");
+    fluid.registerNamespace("gpii.tests.prefs.panel.utils");
```

Comment:
Seems namespaces used in this test file for test panels are inconsistent. We have `gpii.tests.prefs.panel...` and `gpii.tests.firstDiscovery.panel...`. We should pick one to sync up. I lean towards "gpii.tests.firstDiscovery.panel"? What do you think?